src/test-utils.tsx

```diff
@@ -0,0 +1,304 @@
+import {SupportedLocales} from '@open-formulieren/types';
+import {render as rtlRender} from '@testing-library/react';
+import type {RenderOptions, RenderResult} from '@testing-library/react';
+import {Formik} from 'formik';
+import type {FormikErrors, FormikTouched, FormikValues} from 'formik';
+import React from 'react';
+import {IntlProvider} from 'react-intl';
+
+import {BuilderContext} from '@/context';
+import type {ComponentTranslationsRef, DocumentTypeOption, SelectOption} from '@/context';
+import {AnyComponentSchema} from '@/types';
+
+import {PrefillAttributeOption, PrefillPluginOption} from '../src/components/builder/prefill';
+import {RegistrationAttributeOption} from '../src/components/builder/registration/registration-attribute';
+import {ValidatorOption} from '../src/components/builder/validate/validator-select';
+
+const DEFAULT_COMPONENT_TREE: AnyComponentSchema[] = [
```

Comment:
I think it'd be wise to define these constants in place shared between SB & jest? We're still adding some context callables that rely on stuff being put in and having to update two files with the same dummy data is annoying.